file_aloc/bl_import_aloc.py

```
@@ -0,0 +1,292 @@
+    file = open(fp, "rb")
+    br = io_binary.BinaryReader(file)
+    aloc = aloc_format.Physics()
+    aloc.read(filepath)
+    br.close()
+    log("DEBUG", "Finished reading ALOC from file: " + filepath + ".", "read_aloc")
+
+    return aloc
+
+
+def convex_hull(bm):
+    ch = bmesh.ops.convex_hull(bm, input=bm.verts)
```

Ruff (0.14.8)

Local variable `ch` is assigned to but never used

Remove assignment to unused variable `ch`

(F841)